packages/tables/frontend/sirius-components-tables/package.json

```diff
@@ -41,7 +41,8 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
     "tss-react": "4.9.7",
-    "xstate": "4.32.1"
+    "xstate": "4.32.1",
+    "react-router-dom": "6.26.0"
```

Comment:
The table support should not depend on react-router

Reply:
Yes I need the extension point in order to remove this dependency. 
https://github.com/eclipse-sirius/sirius-web/pull/4179